pycqed/analysis_v2/pytomo.py

```diff
@@ -0,0 +1,909 @@
+from cmath import *
+import re, os, math, sys, time, subprocess, string, getopt
+from numpy import zeros, eye
+import uuid
```

Comment:
I think it would be a good idea to add 2 things to this file. 
- move it to a subfolder for everything relating to tomo. 
- add a comment on the top of this file what the purpose of this file is (I assume generic tomography helpers).